docs/references/users/update-user-status.md

```diff
@@ -1 +1 @@
-Update the user status by its unique ID. Use this endpoint as an alternative to deleting a user if you want to keep user's ID reserved.
\ No newline at end of file
+Update the user status by its unique ID. This endpoint deletes all active sessions when a user is blocked, resolving issues when a different user attempts to sign in on the same device.
```

Comment:
Looks like part of the old text was removed. Would you please put that back? 

Also, no need for the part about resolving.

```suggestion
Update the user status by its unique ID. Use this endpoint as an alternative to deleting a user if you want to keep user's ID reserved.

When a user is blocked, all their sessions will also be deleted.
```